Client Scripts/Use case of addOption and removeOption for priority/script.js

```diff
@@ -0,0 +1,10 @@
+function onChange(control, oldValue, newValue, isLoading, isTemplate) {
+   if (isLoading || newValue === '') {
+	return;
+   }
+   if(newValue == 1) {
+	g_form.removeOption('impact',3); // 3 is the value for impact 'low'
+   } else {
+	g_form.addOption('impact', '3 - Low');
```

Comment:
This invocation will add a choice with an empty display value. Please update this to follow the documented way of leveraging this API (`g_form.addOption(field,value,displayValue)` - optionally you could include the fourth parameter for index.